js/_website/src/routes/[[version]]/docs/no-reload/+page.server.ts

```diff
@@ -0,0 +1,72 @@
+import Prism from "prismjs";
+import "prismjs/components/prism-python";
+import { make_slug_processor } from "$lib/utils";
+
+let language = "python";
+
+const COLOR_SETS = [
+	["from-green-100", "to-green-50"],
+	["from-yellow-100", "to-yellow-50"],
+	["from-red-100", "to-red-50"],
+	["from-blue-100", "to-blue-50"],
+	["from-pink-100", "to-pink-50"],
+	["from-purple-100", "to-purple-50"]
+];
+
+export async function load({ parent }) {
+	const {
+		docs,
+		components,
+		helpers,
+		modals,
+		py_client,
+		routes,
+		on_main,
+		wheel
+	} = await parent();
+
+	let headers = [
+		["Description", "description"],
+		["Example Uage", "example-usage"]
+	];
+	let method_headers: string[][] = [];
+	const get_slug = make_slug_processor();
+	let obj = {
+		name: "NO_RELOAD",
+		description:
+			"When launching Any code in a `if gr.NO_RELOAD` code-block will not be re-evaluated when the source file is reloaded. This is helpful for importing modules that do not like to be reloaded (tiktoken, numpy) as well as database connections and long running set up code.",
+		example: `if gr.NO_RELOAD:
```

Comment:
I would include a more complete example of a gradio app here to help distinguish between what should be in `if gr.NO_RELOAD` and what shouldn't